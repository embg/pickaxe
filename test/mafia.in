1 3 5 7
1 3 6 5
2 3 5 10 12 14
42
2
1 6 5 14
1 3 14
1 2 3
