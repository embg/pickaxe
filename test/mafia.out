2 3 (2)
14 1 (2)
14 3 (2)
14 5 (2)
6 5 1 (2)
5 3 1 (2)
